### ISSUE TITLE
6.5.0: attempting to browse to theguardian.com crashes the tab, and it cannot be restored

### ISSUE THREAD

**Issue by IanKemp** [MrAlex94/Waterfox#3604]
### What happened?

As title. Troubleshooting mode (all extensions etc. disabled) makes no difference.

### Reproducible?

- [x] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

G6

### What platform are you seeing the problem on?

Windows

**Comment by thewho2018** [comment]
LOTS of crashes all over... news, tech, youtube, facebook.. Not been a great version for me..

**Comment by Wonderer0** [comment]
Occasional tab crash (< 1 per hour) on various sites, including Google search.  Has been able to recover every time, so far.
Extremely rare in previous versions.

**Comment by MrAlex94** [comment]
Hi all!

Could any of you try the following:

Go to `about:config`, search for `image.jxl.enabled` and set it to `false`?

**Comment by IanKemp** [comment]
@MrAlex94 I can confirm that disabling that flag allows theguardian.com to load successfully.

**Comment by MrAlex94** [comment]
Looks like it was an issue with the JPEG-XL library itself; have updated to the latest available version which stops the crashes :-)

109f77c7a579abb7f500d7a8be2bc10aadea1a8a
fa8ee36bfbcbe7c81037731f23fc4dc1d7f29aa2
b6f88f7dba80dd051bf622a7f2b636abfb9b88a5
ad953bd0f512c2654b216af634e9e94fc0dd2bef

**Comment by MrAlex94** [comment]
Building 6.5.1 which resolves this issue! Please re-open if it continues after.

**Comment by Wonderer0** [comment]
@MrAlex94 Thanks for your very prompt work fixing this :-)

**Comment by Wonderer0** [comment]
Two Google search tabs have just crashed even though `image.jxl.enabled` is `false` 😞

**Comment by Wonderer0** [comment]
Tab crashed again even after upgrading to 6.5.1, with `image.jxl.enabled` still `false`.  It restored immediately without problem.

It only happens occasionally, I had been using it for a total time of at least an hour (i.e. had screen focus) without any problem.  I think it crashed when I went back from a https://engineering.stackexchange.com page to a https://www.google.co.uk search.  I have tried repeating this movement (click link in search ; click back button) several times without problem.

This seems to a different issue to the JPEG-XL one.  Should I open it as a new one?
